ScriptureRenderingPipeline/Rendering.cs

```diff
@@ -123,15 +260,15 @@ private static DocxConfig CreateConfig(IQueryCollection query)
 
             if (query.ContainsKey("lineSpacing"))
             {
-                if(double.TryParse(query["lineSpacing"], out double lineSpacing))
+                if (double.TryParse(query["lineSpacing"], out double lineSpacing))
                 {
                     config.lineSpacing = lineSpacing;
                 }
             }
 
             if (query.ContainsKey("direction"))
             {
-                if(query["direction"] == "rtl")
+                if (query["direction"] == "rtl")
                 {
                     config.textDirection = NPOI.OpenXmlFormats.Wordprocessing.ST_TextDirection.tbRl;

```

Comment:
The method for specifying RTL has changed and no longer depends on the details of NPOI.  Set `config.rightToLeft` to `true`.  More details here: https://github.com/WycliffeAssociates/USFMToolsSharp.Renderers.Docx/blob/master/USFMToolsSharp.Renderers.Docx/DocxConfig.cs

Reply:
Done